### PR TITLE
CI/CD

### PR DESCRIPTION
**Pull request by Jakic007**

### CODE REVIEW

File: scripts/deploy.sh [hunk 1]
```diff
@@ -4,6 +4,8 @@ set -e
 ssh -o "StrictHostKeyChecking no" user@ssh.swip.work "\
```
Comment (edisinovcic) [inline scripts/deploy.sh]: Also add validation for tags.

If CIRCLE_BRANCH is mater, then tag is stable, if branch is develop, then tag is latest.

File: docker-compose.yaml [hunk 2]
```diff
@@ -3,9 +3,6 @@ services:
   swip: 
     image: shardlabs/swip:latest
```
Comment (edisinovcic) [inline docker-compose.yaml]: Change:

> image: shardlabs/swip:latest


to

> image: shardlabs/swip:$TAG

Comment (edisinovcic) [reply docker-compose.yaml]: Tag is defined in the deploy.sh script

File: scripts/deploy.sh [hunk 3]
```diff
@@ -1,9 +1,24 @@
 #!/bin/bash
 set -e
 
+# If not staging or master branch are existing
+export TAG="$CIRCLE_BRANCH"
+
+# if [ "$CIRCLE_BRANCH" == "develop" ]; then
```
Comment (edisinovcic) [inline scripts/deploy.sh]: You don't need to comment this. It can be in config. But it's not that important.

Comment (Jakic007) [reply scripts/deploy.sh]: Ok, uncommented.

**Pull request merged**